util/util.go

```diff
@@ -475,3 +482,58 @@ func (o MachinesByCreationTimestamp) Less(i, j int) bool {
 	}
 	return o[i].CreationTimestamp.Before(&o[j].CreationTimestamp)
 }
+
+// AddWatchOnClusterPaused adds a conditional watch to the controlled given as input
+// that sends watch notifications on any create or delete, and only updates
```

Comment:
In other uses we've not triggered notifications on `Delete` is there a specific use case you have in mind where we would want to get an update on Delete for the Cluster?

Reply:
answered above

Reply:
Just to clarify, since we do not have the Create and Delete predicates specified, does that mean those events will be passed to the event handler? I'm trying to understand this line 491 because it seems that right now we are only specifying the update predicate which verifies if a cluster has been unpaused.

Reply:
That's a good question, need to investigate a little more